server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -518,17 +518,36 @@ const orderMutations = {
         include: [
           { association: 'paymentMethod' },
           { association: 'fromCollective', attributes: ['id', 'slug'] },
-          { association: 'collective', attributes: ['id', 'slug'] },
+          { association: 'collective', attributes: ['id', 'slug', 'HostCollectiveId'] },
         ],
       });
 
       // -- Some sanity checks to prevent issues --
-      const paymentMethodIds = uniq(orders.map(order => order.PaymentMethodId).filter(Boolean));
+      const isAddedFund = order =>
+        order.paymentMethod?.service === PAYMENT_METHOD_SERVICE.OPENCOLLECTIVE &&
+        order.paymentMethod.type === PAYMENT_METHOD_TYPE.HOST;
+      const paymentMethodIds = uniq(
+        orders
+          .filter(order => !isAddedFund(order))
+          .map(order => order.PaymentMethodId)
+          .filter(Boolean),
+      );
       const ordersIds = orders.map(order => order.id);
+
       for (const order of orders) {
+        if (fromAccount) {
+          if (isAddedFund) {
```

Comment:
Ops seems there is a problem here actually. With the default value being different. I made a small correction so that both `order.fromCollective.HostCollectiveId` and `fromAccount.HostCollectiveId` has the same default value. Now it's all good. Sorry about that.

Reply:
My comment was about `isAddedFund`

Reply:
Yes I've tested and it works. I am a bit confused here; you've suggested above https://github.com/opencollective/opencollective-api/pull/8470#discussion_r1119855919 this refactoring.

Reply:
My proposal was based on the existing code. I could not check that the existing code was correct, this is your PR and your task.

Reply:
`isAddedFund` is a function and not a variable, using it alone like this doesn't make any sense, it's always truthy.

Reply:
Oh... That makes sense, I was testing different added funds so they all work. But I think I didn't check for other contributions. I've corrected this now. Thanks. 🙏🏽